velox/functions/sparksql/tests/SliceTest.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/lib/Slice.h"
```

Comment:
Do we need this include?

Reply:
Yes, `SliceTestBase` extends `FunctionBaseTest`, which register presto functions only, we need this to register integer slice Function below, see Line:28

Reply:
Got it. Thanks.